evap/contributor/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.conf.urls import url
 
-from evap.contributor.views import *
+from evap.contributor.views import index, profile_edit, course_view, course_edit, course_preview
```

Comment:
Well, I restricted the import to the views only. Everything else would just unnecessarily clutter the module's namespace, as can be seen here ("Unused imports from wildcard import..."): https://landscape.io/github/fsr-itse/EvaP/370/modules/evap/contributor/urls.py#L3